frontend/components/auth/SignUp/SignUpOptionsForm.tsx

```diff
@@ -30,23 +30,20 @@ const SignUpOptionsForm: React.FC<SignUpOptionsFormProps> = ({
 		return setShowSignUp(SignUpEnum.SIGN_UP);
 	};
 
-	// const [ShowLogIn, setShowLogIn] = useState(false);
 	return (
 		<Container direction="column">
 			<Text size="md">
 				The email
 				<Text size="md" fontWeight="medium">
 					{` ${emailName} `}
 				</Text>
```

Comment:
try replace by something like this pls
```suggestion
				<Text size="md" fontWeight="medium" css={{ marginHorizontal: '5px' }}>
					{emailName}
				</Text>
```